appscale/tools/admin_api/version.py

```diff
@@ -76,6 +78,22 @@ def from_yaml(app_yaml):
     version.env_variables = app_yaml.get('env_variables', {})
     version.inbound_services = app_yaml.get('inbound_services', [])
 
+    automatic_scaling = app_yaml.get('automatic_scaling', None)
+    manual_scaling = app_yaml.get('manual_scaling', None)
+    if manual_scaling:
+      try:
+        version.manual_scaling = {'instances': int(manual_scaling['instances'])}
+      except StandardError:
+        raise AppEngineConfigException('Invalid app.yaml: manual_scaling invalid.')
+    elif automatic_scaling:
```

Comment:
It seems this has an unexpected behavior if the user mistakenly defines both.

I would vote to either include all defined parameters in the request (and let the Admin API reject it) or reject it here.

This is minor and shouldn't block, though.